.github/workflows/publish-docker-helm.yml

```diff
@@ -6,6 +6,18 @@ on:
         description: Name of the application in /lighty-applications
         default: lighty-rnc
```

Comment:
defaults for `app-name` & also `image-name` bellow are also redundant now - make them fully required to enter by user without default values

Reply:
done